presto-main/src/main/java/com/facebook/presto/server/testing/TestingPrestoServer.java

```diff
@@ -178,9 +179,29 @@ public TestingPrestoServer(boolean coordinator,
             SqlParserOptions parserOptions,
             List<Module> additionalModules)
             throws Exception
+    {
+        this(coordinator, properties, environment, discoveryUri, parserOptions, additionalModules, Optional.empty());
+    }
+
+    public TestingPrestoServer(boolean coordinator,
```

Comment:
nit: add a newline before `coordinator` parameter

Reply:
This was copied from the other constructor. I added a new line to that one too.